test/performance/tests/conftest.py

```diff
@@ -187,3 +188,44 @@ def instances(
             _generate_inspection_report(h, instance.id)
 
         h.delete_instance(instance.id)
+
+
+@pytest.fixture(scope="session")
+def session_instance(
+    h: harness.Harness, tmp_path_factory: pytest.TempPathFactory, request
+) -> Generator[harness.Instance, None, None]:
+    """Constructs and bootstraps an instance that persists over a test session.
+
+    Bootstraps the instance with all k8sd features enabled to reduce testing time.
+    """
+    LOG.info("Setup node and enable all features")
+
+    tmp_path = tmp_path_factory.mktemp("data")
+    instance = h.new_instance()
+    snap = next(snap_versions(request))
+    util.setup_k8s_snap(instance, tmp_path, snap)
+
+    bootstrap_config_path = "/root/bootstrap-session.yaml"
```

Comment:
Maybe we can make this configurable for other tests?

Reply:
This is a good idea for the integration test in the k8s-snap repo. For k8s-dqlite we do a load-test with a snap that has all the features enabled. I highly doubt we will need a different bootstrap config in the future- I would rather keep this simple for now.